etc/config.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
     <default>
+    <test>
```

Comment:
@jacstn indent pls

Reply:
@jacstn just in case if you missed this comment (saw you pushed new 2 commits recently)